host/webpack.config.js

```diff
@@ -75,7 +75,6 @@ module.exports = {
         },
         {
           '@openmsupply-client/common': {
-            import: path.join(__dirname, '../common'), // '@openmsupply-client/common',
```

Comment:
Removing these lines removed a 600kb bundle file in the dist folder when doing `yarn build`... I am unsure if that bundling would have ever been served and not sure what this line was meant to be doing - really I shouldnt be touching things I don't understand!

Reply:
I'm not sure how that works for you! for me, that won't build
I get `can't resolve '@openmsupply-client/common'...` if I do. In another branch I've also changed this to `import: @openmsupply-client/common/src` and that resolves build errors for me.

Reply:
Weird..! Sorry, made it worse :) Were you planning to do anymore commits on this branch? If you are i'll leave it otherwise I'll revert it!